.ci.yaml

```diff
@@ -4808,7 +4808,7 @@ targets:
     recipe: packaging/packaging
     timeout: 60
     scheduler: release
-    bringup: true
+    bringup: false
```

Comment:
This target has been timing out frequently: https://ci.chromium.org/p/flutter/builders/prod/Linux%20flutter_packaging_test?limit=50, we do not want to simply flip it back.

Reply:
Set to true and filed https://github.com/flutter/flutter/issues/126286

Reply:
Yes, we should treat them the same, whenever the test becomes flaky and caused tree red.